src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/ProductType.cs

```diff
@@ -85,13 +230,34 @@ private static async Task<object> ResolveConnectionAsync(IMediator madiator, IRe
                     .ToList(),
                 PageInfo = new PageInfo()
                 {
-                    HasNextPage = response.Result.TotalCount > (skip + first),
+                    HasNextPage = response.Result.TotalCount > skip + first,
                     HasPreviousPage = skip > 0,
                     StartCursor = skip.ToString(),
                     EndCursor = Math.Min(response.Result.TotalCount, (int)(skip + first)).ToString()
                 },
                 TotalCount = response.Result.TotalCount,
             };
         }
+
+        // TODO: move this to extensions if this needed in future :p
+        private static T Map<T, TU>(T target, TU source)
```

Comment:
Where this code is used and what it is main purpose?

Reply:
We need to remove this and write propper mapper in mapper profile. This code is temporary.